### PR TITLE
Adding timeout limit to session

### PR DESCRIPTION
**Pull request by chStaiger**
One part of the fix for: #37

### CODE REVIEW

File: ibridges/session.py [hunk 1]
```diff
@@ -51,6 +51,7 @@ def __init__(self, irods_env: Optional[dict] = None,
         self._irods_env: dict = irods_env  # type: ignore
         self._irods_env_path = irods_env_path
         self._irods_session = self.connect()
+        self._irods_session.connection_timeout= 10000
```
Comment (dometto) [inline ibridges/session.py]: Styling: `self._irods_session.connection_timeout = 10000` ;-)

**Review by dometto — APPROVED** [review]
Took the liberty of fixing a whitespace styling issue :)

**Comment by maartenschermer** [comment]
Maybe it's a better idea to keep the default timeout as it was, but make it settable when you initiate the session (and also after it's been initiated). Fixing it this high negates the usefulness of having a timeout, I think.

**Comment by chStaiger** [comment]
> Maybe it's a better idea to keep the default timeout as it was, but make it settable when you initiate the session (and also after it's been initiated). Fixing it this high negates the usefulness of having a timeout, I think.

I agree, and I tested t, but it does not solve our problem. We need further investigation. I will close the PR.

**Pull request closed without merging**